devsetup/scripts/gen-edpm-compute-node.sh

```diff
@@ -30,12 +30,12 @@ if [ ${STANDALONE} = "true" ]; then
 fi
 EDPM_COMPUTE_VCPUS=${EDPM_COMPUTE_VCPUS:-2}
 EDPM_COMPUTE_RAM=${EDPM_COMPUTE_RAM:-4}
-EDPM_COMPUTE_DISK_SIZE=${EDPM_COMPUTE_DISK_SIZE:-20}
+EDPM_COMPUTE_DISK_SIZE=${EDPM_COMPUTE_DISK_SIZE:-30}
 EDPM_COMPUTE_NETWORK=${EDPM_COMPUTE_NETWORK:-default}
 EDPM_COMPUTE_NETWORK_TYPE=${EDPM_COMPUTE_NETWORK_TYPE:-network}
 DATAPLANE_DNS_SERVER=${DATAPLANE_DNS_SERVER:-192.168.122.1}
-CENTOS_9_STREAM_URL=${CENTOS_9_STREAM_URL:-"https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-GenericCloud-9-latest.x86_64.qcow2"}
-BASE_DISK_FILENAME=${BASE_DISK_FILENAME:-"centos-9-stream-base.qcow2"}
+EDPM_IMAGE=${EDPM_IMAGE:-"quay.io/podified-antelope-centos9/edpm-hardened-uefi:current-podified"}
```

Comment:
would this not prevent use form testing with vanila centos locallly.

have you tested this change with the old image?